frontend/src/common/entities.ts

```diff
@@ -43,7 +43,7 @@ export interface Dataset {
   // ethnicity_ontology: string;
   // organism: string;
   // organism_ontology: string;
-  title: string;
+  name: string;
```

Comment:
@Bento007 something for tomorrow 😄 

I remember API doc says `title` but the actual data is `name`:

https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/config/corpora-api.yml#L604

Should I update the `yml` file to `name`? Thank you!